LongWarOfTheChosen/Localization/XComGame.rus

```diff
@@ -4666,3 +4666,281 @@ TacticalText="<Bullet/> Плазмобластер выпускает разру
+BriefSummary="Промахи из основного оружия могут стать царапиной с 25% шансом."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+[MissDamageUpgrade_Sup X2WeaponUpgradeTemplate]
+FriendlyName="Элитная рукоятка"
+FriendlyNamePlural="Элитные рукоятки"
+BriefSummary="Промахи из основного оружия могут стать царапиной с 30% шансом."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+; End Translated
+
+;LWOTC needs translation
+
+[CritUpgrade_Bsc X2WeaponUpgradeTemplate]
+BriefSummary="Increase critical hit chance by +<Ability:CRITINCREASE/>%."
+
+[CritUpgrade_Adv X2WeaponUpgradeTemplate]
+BriefSummary="Increase critical hit chance by +<Ability:CRITINCREASE/>%."
+
+; End Translation
+
+
+; LWOTC needs translation
+BriefSummary="Увеличивает шанс крита на +<Ability:CRITINCREASE/>%."
+; End translation
+
+
+;LWOTC needs Translation
+
+[BasicStockProject X2TechTemplate]
+DisplayName="Приклад"
+Summary="Мы можем разобрать и реконструировать приклад для массового производства. Любой оперативник с оружием оборудованным прикладом имеет больше шансов попасть царапиной"
+
+[AdvancedStockProject X2TechTemplate]
+DisplayName="Улучшенные приклады"
+Summary="Позволяет массово производить улучшенные приклады."
+
+[SuperiorStockProject X2TechTemplate]
+DisplayName="Элитные приклады"
+Summary="Позволяет производить элитные приклады, которые ещё больше увеличивают шансы поцарапать при промахе."
+
+;End Translation
+
+
+; LWOTC Needs Translation
+QuoteText="\Инопланетяне начали адаптироваться к нашей среде. Мы должны расширить наши возможности.\""
+; End Translation
+
+
+; LWOTC Needs Translation
+LocFriendlyName="Battlemaster"
+LocLongDescription="Заступники элитные солдаты, которые ранее были в Адвенте. У них есть доп.слот для гранаты, бонус к точности и защите при стрельбе сверху и меньшее время проникновения на задание."
+LocHelpText="Заступники элитные солдаты, которые ранее были в Адвенте. У них есть доп.слот для гранаты, бонус к точности при стрельбе сверху и меньшее время проникновения на задание."
+LocPromotionPopupText="<Bullet/> В доп.слот для гранат может быть экипирована только граната.<br/><Bullet/> С большей высотые получает бонус к точности и защите.<br/><Bullet/> Этот солдат уменьшает время проникновения на задания."
+
+[SkirmisherGrapple X2AbilityTemplate]
+LocPromotionPopupText="<Bullet/> Крюк позволяет заступникам забираться на препятсвия не тратя действие.<br/><Bullet/> Крюк имеет 2 хода перезарядки (1 ход, если есть паркур)."
+; End Translation
+
+
+; LWOTC Needs Translation
+LocPromotionPopupText="<Bullet/> Столб будет длиться столько ходов, сколько было концентрации у Храмовника.<br/><Bullet/> Столб не завершает ход Храмовника.<br/><Bullet/> Может быть использован после разрыва.<br/><Bullet/> Столб имеет <Ability:SelfCooldown/> ходов перезарядки.<br/><Bullet/> Свободное действие."
```

Comment:
Is this for Skirmisher Grapple?

Reply:
It was templar cover ability

Reply:
Pillar

Reply:
OK, I see you have already translated the text for the Pillar ability below, so please remove this line.